README.md

```diff
@@ -151,16 +151,18 @@ Then you can enable the program in your `configuration.nix` file:
   # Enable Asus Numpad Service
   services.asus-numberpad-driver = {
     enable = true;
+    layout = "up5401ea";
     wayland = true;
     runtimeDir = "/run/user/1000/";
-    layout = "up5401ea";
+    waylandDisplay = "wayland-0"
     config = {
-      "top_left_icon_brightness_func_max_min_only" = "1";
+      "default_backlight_level" = "0x41";
```

Comment:
@scientiac 

You can not set up `default_backlight_level` without setting up `backlight_levels`.

```| `default_backlight_level`                     |          | `0x01`            | default backlight level in hex format `0x00` (must be the value from layout `backlight_levels` or value for disabled ```

Reply:
huh, It worked when i set it. I have set only that option on my device and it defaults to lowest brightness. When otherwise it defaulted to the highest brightness.

Is it " I shouldn't" or "I can't"?

Reply:
@scientiac It can not be done generally like this (or recommended because the code is only in `Readme.md`). Because `backlight_levels` is taken from the layout (each layout represents the hardware model of NumberPad), check `up5401ea.py`. This is why it works for you, but other models may not have multiple backlight levels. So, the configuration values are for further customization. I try to find out which default settings are the most convenient for the majority. This is not the case.

Reply:
@scientiac Does exist github badge for be NixOS compatible?

Reply:
Do you mean this:

![Nix Flakes: Compatible](https://img.shields.io/badge/Nix%20Flakes-Compatible-brightgreen)

Reply:
there is this as well:

https://img.shields.io/badge/NixOS-5277C3?style=for-the-badge&logo=nixos&logoColor=white

Reply:
@scientiac Thank you. I have added. Could you please look at the fixating of version `xkbcommon<1.1`  for NixOS?

Reply:
Working on it!